jetty-ee10/jetty-ee10-servlet/src/main/java/org/eclipse/jetty/ee10/servlet/ContentProducer.java

```diff
@@ -13,12 +13,12 @@
 
 package org.eclipse.jetty.ee10.servlet;
 
-import org.eclipse.jetty.server.Content;
+import org.eclipse.jetty.io.Content;
 import org.eclipse.jetty.util.component.Destroyable;
 import org.eclipse.jetty.util.thread.AutoLock;
 
 /**
- * ContentProducer is the bridge between {@link HttpInput} and {@link org.eclipse.jetty.server.Content.Reader}.
+ * ContentProducer is the bridge between {@link HttpInput} and {@link Content.Source}.
  */
 public interface ContentProducer
```

Comment:
Now that this class isn't dealing with `Content`s but `Chunk`s this class should be renamed.